src/common/actions/auth-store.js

```diff
@@ -228,7 +228,14 @@ async function fetchAccountInfo(root, discharge) {
     headers: { 'Accept': 'application/json' }
   });
   if (response.status >= 200 && response.status < 300) {
-    return { signedAgreement: true, hasShortNamespace: true };
+    const json = await response.json();
```

Comment:
You could save a tiny bit of duplication by moving this above the `if` block, since it's the first line in both branches.